src/Orchard.Cms.Web/Modules/Orchard.OpenId/JwtBearerConfigureOptions.cs

```diff
@@ -33,7 +33,7 @@ public void Configure(JwtBearerOptions jwtBearerOptions)
                 jwtBearerOptions.AutomaticAuthenticate = true;
                 jwtBearerOptions.AutomaticChallenge = true;
                 jwtBearerOptions.RequireHttpsMetadata = !openIdSettings.TestingModeEnabled;
-                jwtBearerOptions.Audience = openIdSettings.Audience;
+                jwtBearerOptions.Audience = openIdSettings.Audiences!=null?string.Join(",",openIdSettings.Audiences):null;
```

Comment:
Hum, are you sure it works? I doubt IdentityModel will split the audiences for you.

Instead, consider using `jwtBearerOptions.TokenValidationParameters.ValidAudiences`.